WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -58,13 +58,15 @@ final class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	/**
 	 * Minimal number of characters the prefix needs in order to be valid.
 	 *
+	 * @since 3.2.0
 	 * @since 2.2.0
```

Comment:
```suggestion
	 * @since 2.2.0
	 * @since 3.2.0 The minimum number of characters was raised from 3 to 4.
```

- Keep the since tags in order from top to bottom (then the history reads in a more logical order). 
- Add the description of what was changed in the latest amendment.

Reply:
Thanks Gary